build.gradle.kts

```diff
@@ -43,15 +43,26 @@ val shouldRunAllCheckers by extra(isCI || project.hasProperty("runAllCheckers"))
 tasks.register<UpdateEapBuildNumber>("updateEapBuildNumber")
 
 tasks.withType<DependencyUpdatesTask> {
-  val isStableVersion: (String) -> Boolean = { version ->
+  fun isStableVersion(version: String): Boolean {
     val stableKeyword = listOf("RELEASE", "FINAL", "GA").any { version.toUpperCase().contains(it) }
-    val regex = "/^[0-9,.v-]+(-r)?$/".toRegex()
```

Comment:
Whoops `/.../` are an equivalent of `toRegex()` in Groovy (syntax taken from Perl); they weren't supposed to be a part of the string regex itself